test/functional/pdo_sqlsrv/pdostatement_fetchmode_emulate_prepare.phpt

```diff
@@ -114,7 +112,6 @@ try {
     } else {
         // more and less than operators do not work for encrypted columns
         $tsql = "SELECT * FROM [$tableName] WHERE NOT ID = :id";
```

Comment:
This else block is necessary, I understand, because comparisons are not supported